staging/src/k8s.io/apiserver/pkg/authentication/authenticatorfactory/delegating.go

```diff
@@ -79,7 +85,13 @@ func (c DelegatingAuthenticatorConfig) New() (authenticator.Request, *spec.Secur
 	}
 
 	if c.TokenAccessReviewClient != nil {
-		tokenAuth, err := webhooktoken.NewFromInterface(c.TokenAccessReviewClient, c.APIAudiences)
+		// Provide a default if WebhookRetryBackoff has not been set by the user.
+		retryBackoff := c.WebhookRetryBackoff
+		if retryBackoff == nil {
+			retryBackoff = webhooktoken.DefaultRetryBackoff()
```

Comment:
Is this being left empty somewhere?  I would expect the options to already have defaults set so this is unnecessary.

Reply:
@deads2k yes, somewhere it's left empty. If I don't default here, all the `kind` tests in ci fail. We are throwing error now. So I am hoping I can find the spot.